src/legacy/core_plugins/kibana/public/dashboard/store/panel_actions_store.ts

```diff
@@ -28,7 +28,9 @@ class PanelActionsStore {
    */
   public initializeFromRegistry(panelActionsRegistry: ContextMenuAction[]) {
     panelActionsRegistry.forEach(panelAction => {
```

Comment:
I noticed that in navigating to the dashboard several times (wo a full page reload), my "Export" panel action kept gettin added to panel actions list, so there was numerous duplicates. AFAICT this is the only module so far that uses the panel-action registry, so could have been a bug? Either way this fixes it.

Reply:
That looks like a bug fixto me!